2.-Configuration.md

```diff
@@ -50,6 +50,11 @@ We can disable the logging for all your services by using `-Dts.global.log.enabl
 
 By default logs are using ANSI colors, use `-Dts.global.log.nocolor=true` to disable them.
 
+# Service directory
+
+Some services crate their own folder under the `target` directory, which will contain temporary files (e.g. CLI or DEV mode services).
```

Comment:
```suggestion
Some services create their own folder under the `target` directory, which will contain temporary files (e.g. CLI or DEV mode services).
```
and (optionally!)
```suggestion
Some services (e.g. CLI or DEV mode services) create their own folder under the `target` directory, which will contain temporary files.
```
TBH I don't know what CLI services are, can you give me a hint?

Reply:
Anyway, I think you just mean any `BaseService` impl. and it is alright

Reply:
I rephrased the whole section, so that it talks about services in general. I also corrected the typos and added an example for both global and service-specific config.